org-super-agenda.el

```diff
@@ -437,8 +437,9 @@ match items that have no deadline), `past` (to match items with a
 deadline in the past), `today' (to match items whose deadline is
 today), or `future' (to match items with a deadline in the
 future).  Argument may also be given like `before DATE' or `after
-DATE', where DATE is a date string that
-`org-time-string-to-absolute' can process."
+DATE', where DATE is a date string that `org-read-date' can
+process.  Note that relative dates are supported, e.g. `before
++3d' means in the next two days."
```

Comment:
The `+3d` here should be in a string, right?

Reply:
Yes, do you think that is not clear?

Reply:
The part in the backtick-apostrophe quote indicates Lisp code, i.e. like `(:scheduled (before "+3d"))`.  Without the string, it wouldn't be valid, e.g. `(:scheduled (before +3d))`.  So the `+3d` should appear as a string, I think.